i18n/en.json

```diff
@@ -466,6 +466,12 @@
   "admin.experimental.emailBatchingInterval.desc": "Specify the maximum frequency, in seconds, which the batching job checks for new notifications. Longer batching intervals will increase performance.",
   "admin.experimental.emailBatchingInterval.example": "E.g.: \"30\"",
   "admin.experimental.emailBatchingInterval.title": "Email Batching Interval",
+  "admin.experimental.emailSettingsLoginButtonBorderColor.desc": "Specify the color of the Email login button border for white labeling purposes. Use a hex code with a #-sign before the code. This setting only applies to the mobile apps.",
+  "admin.experimental.emailSettingsLoginButtonBorderColor.title": "Email login Button Border Color",
```

Comment:
`login` > `Login`. Also misses `:` at end of title

Reply:
Aha, interesting -- none of the settings I added to this page had a trailing colon, but in aggregate, I didn't notice!

Curious if we actually need the colons in practice?

👍 re: `Login` casing

Reply:
> Curious if we actually need the colons in practice?

IMO not really, but I'm 0/5. For consistency we should either have them throughout or not at all.

With the System Console reorganization effort, maybe we can revisit this and make a decision on whether to keep the colons or not at the same time.